pkgs/by-name/pl/plumber/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "plumber";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "streamdal";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-ftXLipJQjRdOSNO56rIRfAKKU0kHtAK85hgcT3nYOKA=";
+  };
+
+  vendorHash = null;
+
+  # connection tests create a config file in user home directory
+  preCheck = ''
+    export HOME="$(mktemp -d)"
+  '';
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/streamdal/plumber/options.VERSION=${version}"
+  ];
+
+  meta = with lib; {
+    description = "plumber is a CLI devtool for inspecting, piping, messaging and redirecting data in message systems like Kafka, RabbitMQ, GCP PubSub and more";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
description can not start with package name and perhaps could be shorter.